liota/dcc_comms/mqtt_dcc_comms.py

```diff
@@ -124,17 +125,32 @@ def _disconnect(self):
         """
         self.client.disconnect()
 
-    def subscribe(self, msg_attr=None):
+    def receive(self, msg_attr=None):
```

Comment:
As discussed, in case of auto-generation callback method will be enforced to be used.